docs/essentials/element_attributes.md

```diff
@@ -36,6 +36,37 @@ Besides using values in pixels (i.e. `w="100" h="300"`), you can also specify _p
 The percentage value specified for `w` and `x` will be calculated as the percentage of the _width_ (`w`) of the parent element.
 And the percentage specified for `h` and `y` will use the _height_ (`h`) of the parent element as the base of the percentage calculation.
 
+### Predefined placement options
+
+In addition to the absolute positions that Blits and Lightning use, there are a few predefined placement options available. By adding the `placement`-attribute to an Element we can easily _center_ or align an Element to a corner, without adding custom calculations.
```

Comment:
Would change the second sentence to: By add the `placement`-attribute to an Element we can easily align it to the center, left, right, top, bottom, or even a combination like `{y: 'center', x: 'left'}` for example without having to calculate the positions yourself.